frontend/src/components/RequestDialog.vue

```diff
@@ -1,17 +1,31 @@
 <template>
-  <div class="request-dialog">
-    <RequestFormInputs v-model="formResult" />
-
-    <Teleport v-if="signer" to="#action-button-portal">
-      <ActionButton
-        v-if="transferFundsButtonVisible"
-        :disabled="!formValid"
-        @click="submitRequestTransaction"
-      >
-        Transfer Funds
-      </ActionButton>
-    </Teleport>
+  <div class="request-dialog bg-transparent flex flex-col justify-between h-full">
```

Comment:
Nitpick (just because are working on it right now): the `request-dialog` class is completely unused according to what I see.